go.mod

```diff
@@ -1,68 +1,76 @@
 module github.com/amazeeio/fastly-controller
 
-go 1.18
+go 1.19
 
 require (
 	github.com/fastly/go-fastly v1.18.0
-	github.com/go-logr/logr v1.2.0
-	gopkg.in/robfig/cron.v2 v2.0.0-20150107220207-be2e0b0deed5
```

Comment:
This would downgrade to v1. Could we leave it at v2 or upgrade to v3 (github.com/robfig/cron/v3)?